taskcluster/fpm_taskgraph/transforms/build.py

```diff
@@ -0,0 +1,72 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Base file is https://github.com/mozilla-mobile/fenix/blob/d918cfdf3841a17545053dca2e7480c648ab4f95/taskcluster/fenix_taskgraph/transforms/build.py. I removed what's unnecessary and tweaked the `flavor` thing.